### DISCUSSION TITLE
Security Issue - Why does LDAP / Active Directory keep Admin Bind DN as plain text?

### DISCUSSION THREAD

**Discussion by PureJoyMind** [requarks/wiki#6898] [Error / Bug Report] [answered]
I am using version 2.5.300:
![image](https://github.com/requarks/wiki/assets/27802665/dffb653b-d3cc-41e6-9bff-d09a8a085575)

In the LDAP / Active Directory Authentication strategy, why does the wiki hold the Admin Bind DN as plain text field? isn't this a security issue? 
was this supposed to be handled by having proper Permissions and Page Rules? 
![image](https://github.com/requarks/wiki/assets/27802665/5c2b795b-6eb6-4816-9249-878a5c5ca3a4)

**Comment by NGPixel** [comment, marked as answer]
The admin "Authentication" section is only accessible to the administrator group. A user with such access can already do and access virtually anything.

The password cannot be hashed as it's used directly in the passport-ldap module and must be passed as-is when authenticating with the LDAP server.

The upcoming v3 will "hide" the password from the client view (only allow to be overwritten) but the value itself will still be saved in plaintext in the DB for the same reason as above.

So unless you give anyone administrator access, this is not a security issue.
